backend/src/schema/resolvers/reports.spec.js

```diff
@@ -480,7 +475,6 @@ describe('reports query', () => {
             submitter: expect.objectContaining({
               id: 'user1',
             }),
-            resourceId: 'auth1',
```

Comment:
I was in a hurry and not concentrated … 😬

Reply:
well, it works for queries, but didn't work for mutations, we could easily add it to the mutations as well, I just didn't feel like it's necessary, and didn't hear back from you so I just thought I'd make the change and revert if people wanted me to